packages/plugins/typescript-documents/tests/ts-documents.spec.ts

```diff
@@ -845,5 +845,55 @@ describe('TypeScript Documents Plugin', async () => {
       `)
       );
     });
+
+    it('Should generate correctly when using enums and typesPrefix', async () => {
+      const testSchema = makeExecutableSchema({
+        typeDefs: parse(/* GraphQL */ `
+          enum Access {
+            Read
+            Write
+            All
+          }
+          type User {
+            access: Access
+          }
+          input Filter {
+            match: String!
+          }
+          type Query {
+            users(filter: Filter!): [User]
+          }
+        `)
+      });
+      const query = parse(/* GraphQL */ `
+        query users($filter: Filter!) {
+          users(filter: $filter) {
+            access
+          }
+        }
+      `);
+
+      const content = await plugin(
+        testSchema,
+        [{ filePath: '', content: query }],
+        { typesPrefix: 'PREFIX_' },
+        {
+          outputFile: 'graphql.ts'
+        }
+      );
+
+      // Kamil: is `PREFIX_Prefix_*` correct?
```

Comment:
@dotansimha ?

Reply:
Nope, it should be just `PREFIX_*`. It seems like we apply the prefix twice, one without `convertName` and one with it.

Reply:
I'm on it